recon/modules/v1_osint/hibp_email.py

```
@@ -0,0 +1,55 @@
+        return [{"email": "alice@example.com", "api_key": "YOUR_HIBP_KEY"}]
+
+    def run(self, email: str, api_key: str, http: HTTPClient, **kwargs) -> Dict[str, Any]:
+        url = f"https://haveibeenpwned.com/api/v3/breachedaccount/{quote(email)}"
+        headers = {
+            "hibp-api-key": api_key,
+            "user-agent": "ReconINT/1.0",
+            "accept": "application/json",
+        }
+        try:
+            resp = http.http_get(url, headers=headers)
+        except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)